Cannot find module '@aws-cdk/core/lib/private/uniqueid'

While trying to run `cdk-dia` against a CDK app I get the following error:

```
npx cdk-dia
npx: installed 36 in 3.679s
Cannot find module '@aws-cdk/core/lib/private/uniqueid'
Require stack:
- /Users/matthewbonig/.npm/_npx/23996/lib/node_modules/cdk-dia/src/diagram/aws/aws-edge-resolver.js
- /Users/matthewbonig/.npm/_npx/23996/lib/node_modules/cdk-dia/src/diagram/index.js
- /Users/matthewbonig/.npm/_npx/23996/lib/node_modules/cdk-dia/src/cdk-dia.js
- /Users/matthewbonig/.npm/_npx/23996/lib/node_modules/cdk-dia/bin/cli.js
```

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -43,8 +43,23 @@ jobs:
             cdk init sample-app --language typescript
             npm install cdk-dia@9.9.${GITHUB_RUN_NUMBER} --registry http://localhost:4873
 
-            # Use CLI and test expected PNG
+            ## Use CLI and test expected PNG
             cdk synth
             npx cdk-dia
+            test -s diagram.png && echo "diagram.png exists and has size > 0"
+            ls -la
+
+            # Use as a globally installed package
+            npm install -g typescript
+            npm install -g aws-cdk
+            npm install cdk-dia@9.9.${GITHUB_RUN_NUMBER} --registry http://localhost:4873 -g
+            sudo apt install graphviz
+            mkdir package-smoke-test-global
+            cd package-smoke-test-global
+            cdk init sample-app --language typescript
+
+            ## Use CLI and test expected PNG
+            cdk synth
+            cdk-dia
             test -s diagram.png && echo "diagram.png exists and has size > 0"
             ls -la

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "cdk-dia",
-  "version": "0.0.3",
+  "version": "0.0.4",
   "description": "🎡 Automated diagrams of CDK provisioned infrastructure",
   "scripts": {
     "prepare-dist": "./prepare-dist.sh",
@@ -26,9 +26,7 @@
     "ts-graphviz": "^0.13.2",
     "values.js": "^2.0.0",
     "word-wrap": "^1.2.3",
-    "yargs": "^16.2.0"
-  },
-  "peerDependencies": {
+    "yargs": "^16.2.0",
     "@aws-cdk/core": "^1.40.0"
   },
   "devDependencies": {

